corehq/project_limits/admin.py

```diff
@@ -27,3 +28,9 @@ class PillowThrottleDefinitionAdmin(admin.ModelAdmin):
                     'max_value', 'average_value', 'is_enabled')
     list_filter = ('key', 'is_enabled')
     ordering = ('key',)
+
+
+@admin.register(SystemLimit)
+class SystemLimitAdmin(admin.ModelAdmin):
+    list_display = ('key', 'limit')
```

Comment:
I think it'd be useful to add domain here.

Reply:
[c8dc267](https://github.com/dimagi/commcare-hq/pull/35675/commits/c8dc2676badd5a7c16465349ceabd069879e57e4)